### ISSUE TITLE
Add references and page about accessibility (Toegankelijkheidverklaring & WCAG)

### ISSUE THREAD

**Issue by GerbenBosschieter** [teamforus/forus#967]
## Context/goal: 
One of the demands of our stakeholders is to be compliant with the WCAG guidelines and providing an accessibility statement. 

- [x] WCAG2.0 test
- [x] Accessibility statement
- [ ] Add a page to all webshops
- [ ] add css to this page to make it look nice and conform the webshops design.
- [ ] add a reference (name: toegankelijkheid) to the footer next to the Privacy and Terms of agreement 
- [ ] select all information of the accessibility page that are variables. @GerbenBosschieter 

![example_accesabilty](https://user-images.githubusercontent.com/38419514/69734194-dea8e580-112e-11ea-94fd-fb671534bf9e.jpg)

 @danrminds could you guys start developing this page with the example of the image I uploaded. Like a markup page with all the text hard coded?  I will pinpoint the exact information that are going to be variables.

**Comment by GerbenBosschieter** [comment]
As we currently outsource everything thats code related :D I would like to ask you guys to proceed with this issue.

What needs to be done?
Checkout my branch I started : https://github.com/teamforus/forus-frontend/compare/page-accesibility?expand=1

This is a new page thats should be added to all the webshops. I already entered all the content in it. Pleas make it more readable and conform the common designs.

**Comment by GerbenBosschieter** [comment]
https://github.com/teamforus/forus-frontend/tree/page-accesibility

Please continue with this branch.
What needs to be done ?

- [ ] add css to this page to make it look nice and conform the webshops design.
- [ ] add a reference to the footer next to the Privacy and Terms of agreement

**Comment by jamalv** [comment]
Add variables for signatures per implementation @GerbenBosschieter please clarify

**Comment by jamalv** [comment]
@GerbenBosschieter could you share the text in the image as editable text so that it is easier to copy-paste?

**Comment by GerbenBosschieter** [comment]
@jamalv about the text. I will.

About the variables:
The page about accessibility needs to have a signature from a board member. We discussed this with some clients and they want to have their own board member to sign the accessibility page. So that means that al these pages will have different board member names and signatures.

Would be good for now to just have a page with plain text ready so we can continue the conversations with the clients and don't have to wait developing this page after they gave their credentials.

**Comment by GerbenBosschieter** [comment]
Text:


### Toegankelijkheidsverklaring

Westerkwartier streeft naar het toegankelijk maken van de eigen online informatie en dienstverlening, in overeenstemming met Tijdelijk besluit digitale toegankelijkheid overheid.

Deze toegankelijkheidsverklaring is van toepassing op de inhoud van de website Westerkwartier webshop die valt binnen de werkingssfeer van Tijdelijk besluit digitale toegankelijkheid overheid.

De links waarop de inhoud van de website Westerkwartier webshop te vinden is:

- het hoofddomein:
    - [https://westerkwartier.forus.io/](https://westerkwartier.forus.io/)

- subdomeinen en andere domeinen die tot de website behoren:
    - [https://westerkwartier.forus.io/products](https://westerkwartier.forus.io/products)
    - [https://westerkwartier.forus.io/vouchers](https://westerkwartier.forus.io/vouchers)
    - [https://westerkwartier.forus.io/provider](https://westerkwartier.forus.io/provider)
    - [https://westerkwartier.forus.io/sponsor](https://westerkwartier.forus.io/sponsor)

Een actueel en volledig overzicht van de toegankelijkheidsverklaringen die vallen onder de verantwoordelijkheid van Westerkwartier is beschikbaar via de volgende link:

- [link nog niet beschikbaar; moet pas in stap 9 worden ingevuld]

#### Nalevingsstatus

Westerkwartier verklaart dat de website Westerkwartier webshop gedeeltelijk voldoet aan Tijdelijk besluit digitale toegankelijkheid overheid.

Uit toegankelijkheidsonderzoek is gebleken dat nog niet aan alle eisen wordt voldaan. Voor elke afzonderlijke afwijking van de eisen is de oorzaak bekend en is het gevolg beschreven, zijn maatregelen genomen om de afwijking te kunnen opheffen en is een concrete planning gemaakt waarop de maatregelen zullen zijn uitgevoerd.

Zie onder het kopje [Toelichting op de nalevingsstatus](#toelichting-op-de-nalevingsstatus) voor meer gedetailleerde informatie.

##### Opstelling van deze verklaring

De actualiteit, volledigheid en juistheid van deze verklaring zijn voor het laatst herzien op 31-10-2019.

##### Feedback en contactgegevens

Loopt u tegen een toegankelijkheidsprobleem aan? Of heeft u een vraag of opmerking over toegankelijkheid?  
Neem dan contact op via [[toegankelijkheid@naam-overheidsinstantie.nl](mailto:toegankelijkheid@naam-overheidsinstantie.nl)].

##### Wat kunt u van ons verwachten?

- Binnen 5 werkdagen krijgt u een ontvangstbevestiging.
- We informeren u over de voortgang en de uitkomst.
- Binnen 3 weken is uw verzoek afgehandeld.

##### Handhavingsprocedure

Bent u niet tevreden met de manier waarop uw klacht is behandeld? Of hebben we niet op tijd gereageerd?  
Dan kunt u [[contact opnemen met de Nationale Ombudsman](https://www.nationaleombudsman.nl/klacht-indienen/uw-klacht)].

#### Toelichting op de nalevingsstatus

In deze paragraaf wordt de claim dat **_gedeeltelijk_** aan de in Tijdelijk besluit digitale toegankelijkheid overheid gestelde eisen is voldaan nader onderbouwd.

De website Westerkwartier webshop is onderzocht op toegankelijkheid en uit de rapportage blijkt volgens Westerkwartier dat aan _alle_ onderstaande kenmerken is voldaan:

1. het onderzoek omvat alle inhoud die volgens Tijdelijk besluit digitale toegankelijkheid overheid toegankelijk moet worden gemaakt;
2. het onderzoek is gebaseerd op meet- en onderzoeksgegevens die niet ouder zijn dan 12 maanden;
3. de handmatige evaluatie is uitgevoerd overeenkomstig een adequaat gedocumenteerde evaluatiemethode; [

WCAG-EM](https://w3.org/TR/WCAG-EM/) of gelijkwaardig;
4. voor de (semi-)automatische tests is een toetsinstrument ingezet dat is gebaseerd op de algoritmen die zijn gedocumenteerd door de [Auto-

WCAG](https://act-rules.github.io/pages/about/) Community Group;
5. alle onderzoekresultaten zijn nauwkeurig, eenduidig en op reproduceerbare wijze vastgelegd in een voor mensen leesbaar formaat **_OF_** in het machineleesbare formaat [

EARL](https://www.w3.org/WAI/standards-guidelines/earl/);
6. voor elke afzonderlijke afwijking op de eisen die tijdens het onderzoek werd gevonden en die niet kon worden hersteld wordt aangegeven:
    - [referentienummer]
        - [beknopte beschrijving van de afwijking]
        - **oorzaak:** [reden waarom (nog) niet aan de eis kon worden voldaan]
        - **gevolg:** [impact van de afwijking voor personen met een functiebeperking]
        - **alternatief:** [of een toegankelijk alternatief beschikbaar is. En zo ja, welk]
        - **maatregel:** [te nemen maatregel(en) om de afwijking op te heffen]
            - mogelijkheid om aan te geven of de uitvoering van de maatregel een [_onevenredige last_](#onevenredige-last) met zich meebrengt: [ja/nee] [toelichting als het antwoord op de vraag 'ja' is]

        - **planning:** [uiterste datum waarop de afwijking zal zijn hersteld]

(zie onder het kopje [Afwijkingen](#afwijkingen-voldoet-gedeeltelijk))
7. alle evaluatie- en onderzoekresultaten waarop de claim is gebaseerd zijn online beschikbaar  
(zie onder het kopje [Evaluatie- en onderzoekresultaten](#evaluatie-en-onderzoekresultaten-voldoet-gedeeltelijk)

##### Afwijkingen

1. 1.1.1 (= 

SC uit 1.1.1 - Niet-tekstuele content [niveau A])
    - Een img-element mist een alt-attribuut. (alternatieve tekst voor de afbeelding)
    - **oorzaak:** Geen alternatieve tekst toegevoegd bij de referentie van de afbeelding.
    - **gevolg:** Het missen van een alternatieve tekst kan invloed hebben op de functionaliteit van spraak naar tekst hulpmiddelen.
    - **alternatief:** Er is geen alternatieve oplossing voor dit probleem.
    - **maatregel:** De webmasters zullen een alternatieve tekst plaatsen bij de afbeelding.
        - brengt de uitvoering van de maatregel een [_onevenredige last_](#onevenredige-last) met zich mee? **Nee**  

    - **planning:** 01-01-2020

2. 4.1.2 (= 

SC uit 4.1.2 - Naam, rol, waarde [niveau A])
    - Ankerelement gevonden met een geldig href-attribuut, maar er is geen linkinhoud is opgegeven.
    - **oorzaak:** Geen consequent gebruik van ankerelementen
    - **gevolg:** Er is geen direct nadelig gevolg voor de gebruiker.
    - **alternatief:** Er is geen alternatieve oplossing voor dit probleem.
    - **maatregel:** De webmaster zullen het ankerelement opnieuw beoordelen en deze verwijderen of de benodigde informatie opgeven
        - brengt de uitvoering van de maatregel een [_onevenredige last_](#onevenredige-last) met zich mee? **Nee**  

    - **planning:** 01-01-2020

##### Evaluatie- en onderzoekresultaten

- https://wcag.nl/rapporten/a285354cb8b1/
    - onderzoeksdatum: 31-10-2019
    - onderzoeksmethode: automatisch onderzoek
    - de beschikbare informatie is gebaseerd op meet- en onderzoeksgegevens die **niet** ouder zijn dan 12 maanden
    - de handmatige evaluatie is uitgevoerd overeenkomstig een adequaat gedocumenteerde evaluatiemethode; [

WCAG-EM ](https://w3.org/TR/WCAG-EM/)of gelijkwaardig: **Onbekend**

##### Inhoud die buiten de werkingssfeer valt van Tijdelijk besluit digitale toegankelijkheid overheid

Zie [Artikel 2, tweede lid van het Tijdelijk besluit digitale toegankelijkheid overheid](https://zoek.officielebekendmakingen.nl/stb-2018-141.html#d17e165).

**Comment by linda-90** [comment]
Toegankelijkheidsverklaringen gedeeld met:

- [ ] SDOA (berkelland, oost-gelre en winterswijk)
- [ ] Westerkwartier
- [ ] Nijmegen
- [ ] Noordoostpolder

**Comment by jamalv** [comment]
Maybe even just start with a PDF? @GerbenBosschieter what do you think?

**Comment by jamalv** [comment]
Answer; it cannot be a pdf. Please add this page as markdown.

Preferably as
westerkwartier.forus.io/toegankelijkheidsverklaring
nijmegen.forus.io/toegankelijkheidsverklaring
etc.

with a link in the footer

Algemene Voorwaarden | Privacy Statement | Toegankelijkheidsverklaring

**Comment by linda-90** [comment]
@jamalv 
Or
westerkwartier.forus.io/toegankelijkheid
nijmegen.forus.io/toegankelijkheid
etc.

**Comment by linda-90** [comment]
Definitieve verklaringen:
- [ ] Nijmegen
- [ ] Noordoostpolder
- [ ] Westerkwartier
- [ ] SDOA

**Comment by linda-90** [comment]
Wanneer wordt dit doorgevoerd?

**Comment by maxvisser** [comment]
noordoostpolder and SDOA still needs variables.

**Comment by linda-90** [comment]
Nog geen reactie van Noordoostpolder en SDOA. Verwacht ik deze week.

**Comment by GerbenBosschieter** [comment]
@maxvisser Het missing van de variables mag geen reden zijn om niet te beginnen met de toegankelijkheidsverklaring. Dit is ook al vaker aangegeven

**Comment by maxvisser** [comment]
Pug page is almost done; as some implementations still needs variables we will wait for this until it can be added to develop.

**Comment by linda-90** [comment]
Toegankelijkheidsverklaring van Noordoostpolder stellen ze uit tot september 2020.

**Comment by jamalv** [comment]
@GerbenBosschieter @maxvisser @aghimpu what is the status of this issue?

**Comment by maxvisser** [comment]
there are some problems with the pages for SDOA; it results in alot of extra code as we did for home.pug; The platform has no definition of Social Dienst Oost Achterhoek (SDOA) so we need some way to add their name to texts as variables or you are copy pasting text for each SDOA implementation or creating a seperate pug file where you need to somehow check for Berkelland, Oost-gelre and winterswijk to show a different Pug/i18n translation.

These solution all feel pretty hacky. please check https://github.com/teamforus/implementation/issues/169

**Comment by jamalv** [comment]
Nice to have: release on february 3rd, must have feb. 10th.
